packages/babel-helpers/src/helpers.js

```diff
@@ -75,9 +75,9 @@ helpers.asyncToGenerator = template(`
             resolve(value);
           } else {
             return Promise.resolve(value).then(function (value) {
```

Comment:
Not sure how we would write the equivalent test but do you want to PR that @jridgewell?